modules/ROOT/pages/anypoint-monitoring-visualizer.adoc

```diff
@@ -0,0 +1,180 @@
+= Anypoint Monitoring and Visualizer
+
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+
+== Overall Architecture
+
+Anypoint Monitoring is an optional feature during installation, which includes Anypoint Visualizer. Both products require 3 dedicated nodes, normally labeled *amv_node*.
+
+In those 3 nodes, the following Helm charts will be installed in 4 different namespaces:
+
+|===
+|Namespace | Service | Notes
+
+.6+|dias
+|prov-k8s-am-influxdb-comp
```

Comment:
This part describing the services should be moved to the runbook

Reply:
Done. Forgot to push